test/suite/test_export01.py

```diff
@@ -0,0 +1,151 @@
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+#
+
+from helper import copy_wiredtiger_home
+from helper_tiered import TieredConfigMixin, tiered_storage_sources
+from wtscenario import make_scenarios
+import os, wttest
+
+class test_export01(TieredConfigMixin, wttest.WiredTigerTestCase):
+    dir = 'backup.dir'
+
+    types = [
+        ('table', dict(type = 'table:')),
+    ]
+
+    scenarios = make_scenarios(tiered_storage_sources, types)
+
+    # Test that the WiredTiger.export file is correctly created and removed.
+    def test_export(self):
+        uri_a = self.type + "exporta"
+        uri_b = self.type + "exportb"
+        uri_c = self.type + "exportc"
+
+        # Create a few tables.
+        self.session.create(uri_a)
+        self.session.create(uri_b)
+        self.session.create(uri_c)
+
+        # Insert some records.
+        c1 = self.session.open_cursor(uri_a)
+        c1["k1"] = "v1"
+        c1.close()
+
+        c2 = self.session.open_cursor(uri_b)
+        c2["k2"] = "v2"
+        c2.close()
+
+        c3 = self.session.open_cursor(uri_c)
+        c3["k3"] = "v3"
+        c3.close()
+
+        self.session.checkpoint()
+
+        if self.is_tiered_scenario():
+            self.session.flush_tier(None)
+
+        # Open a special backup cursor for export operation.
+        export_cursor = self.session.open_cursor('backup:export', None, None)
+
+        os.mkdir(self.dir)
+        copy_wiredtiger_home(self, '.', self.dir)
+        self.assertTrue(os.path.isfile("WiredTiger.export"))
+
+        export_cursor.close()
+
+        # The export file should be removed from the home directory.
+        self.assertFalse(os.path.isfile("WiredTiger.export"))
+        
+        # The export file should exist in the backup directory.
+        self.assertTrue(os.path.isfile(self.dir + "/WiredTiger.export"))
+
+    def test_export_restart(self):
+        uri_d = self.type + "exportd"
+        uri_e = self.type + "exporte"
+        uri_f = self.type + "exportf"
+
+        # Create two tables.
+        self.session.create(uri_d)
+        self.session.create(uri_e)
+
+        # Insert some records.
+        c4 = self.session.open_cursor(uri_d)
+        c4["k4"] = "v4"
+        c4.close()
+
+        c5 = self.session.open_cursor(uri_e)
+        c5["k5"] = "v5"
+        c5.close()
+
+        self.session.checkpoint()
+
+        if self.is_tiered_scenario():
+            self.session.flush_tier(None)
+
+        # Open a special backup cursor for export operation.
+        main_cursor = self.session.open_cursor('backup:export', None, None)
+
+        # Copy the main database to another directory, including the WiredTiger.export file.
+        os.mkdir(self.dir)
+        copy_wiredtiger_home(self, '.', self.dir)
+
+        main_cursor.close()
+        self.close_conn()
+
+        # Open a connection and session on the directory copy.
+        self.conn = self.setUpConnectionOpen(self.dir)
```

Comment:
Personally I would like to use `self.wiredtiger_open` instead, to stay the same with the other backup tests.

Reply:
Discussed with Jie about this, we can't do this because the bucket directory is not copied properly into the backup directory.